.github/workflows/run-benchmarks.yml

```diff
@@ -0,0 +1,43 @@
+name: Run benchmarks
+# ToDo: add an arbitrary label to keep this cache separate from the builds
+on:
+  workflow_dispatch: # Manually triggered by an authorized repo contributor.
+  pull_request:
+    paths:
+      - '.github/workflows/run-benchmarks.yml'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.head.label || github.head_ref || github.ref }}
+  cancel-in-progress: true      
+jobs:
+  benchmark:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest-8-cores
+    strategy:
+      matrix:
+        runtimes: [centrifuge, altair]
```

Comment:
I think now Demo uses development we should add also here the weights. See this [slack thread](https://kflabs.slack.com/archives/C04GJQAM9P0/p1700118738223449)